Packs/SentinelOne/Integrations/SentinelOne-V2/SentinelOne-V2.py

```diff
@@ -1146,28 +1154,29 @@ def get_threats_command(client: Client, args: dict) -> CommandResults:
 
 def get_hash_command(client: Client, args: dict) -> CommandResults:
     """
-    Get hash reputation.
+    Get hash verdict.
```

Comment:
As suggested in the demo, this command be replaced to use `!file` reputation command. See https://xsoar.pan.dev/docs/integrations/generic-commands-reputation and 
https://xsoar.pan.dev/docs/integrations/context-and-outputs#return-filehash-reputation for more details.